src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx

```diff
@@ -59,8 +59,9 @@ function SubtestsBreadcrumbs({ view }: SubtestsBreadcrumbsProps) {
         marginBottom: 3,
       }}
     >
-      [<></>,<Link href='/'>Home</Link>,
-      <Link href={allResultsURL}>All results</Link>, ]
```

Comment:
I think that when this was implemented Beatrice tried to use an array (with the `[]`), but this doesn't work as expected here: indeed this is not an array (because it should be in `{}`), here the brackets are just handled as normal text and then filtered out by the Breadcrumb component in https://github.com/mui/material-ui/blob/17026fc203a8b413ec7343b6eeb267a50519d515/packages/mui-material/src/Breadcrumbs/Breadcrumbs.js#L174

The main issue here is the fragment as the first item. This is done this way so that there's a separator already before the `Home` item.

Instead I'm inserting an empty `span`. I don't love it but the check that I mentioned above wants a real React component. An alternative could be to create a component that just returns `null`... but this would still generate an empty `li`, so I thought a `span` is just as good.